docs/installation/server-config/index.rst

```diff
@@ -195,53 +200,50 @@ The default number of partitions for the topics created by KSQL. The default is
 ksql.sink.replicas
 ^^^^^^^^^^^^^^^^^^
 
-The default number of replicas for the topics created by KSQL. The default is one. 
+The default number of replicas for the topics created by KSQL. The default is one.
 
 .. _ksql-listeners:
 
 ^^^^^^^^^
 listeners
 ^^^^^^^^^
 
-Set the port for the KSQL Server to listen on. This defaults to http://localhost:8088. To listen for on port 80, update it accordingly
-in your ``ksql-server.properties`` file.
+Set the port for the KSQL Server REST API to listen on. This defaults to http://localhost:8088. To listen on port ``80``, update it accordingly in your ``ksql-server.properties`` file.
 
 .. code:: bash
 
     # Set which port to listen on.
-    listeners=http://0.0.0.0:80 
-    
-.. _restrict-ksql-interactive:
+    listeners=http://0.0.0.0:80
 
-----------------------------------
-Restricting Interactive KSQL Usage
-----------------------------------
+.. _restrict-ksql-interactive:
 
-KSQL supports locked-down deployment scenarios where you can restrict interactive use of the KSQL cluster.
+------------------------------------
+Headless, non-interactive KSQL Usage
+------------------------------------
 
-You can prevent interactive use of a KSQL cluster. For example, you want to allow a team of users to develop and verify
-their queries on a shared testing KSQL cluster. But when putting those queries to production you prefer to lock-down access
-to KSQL Servers, version-control the exact queries and storing them in a .sql file, and prevent users from interacting
-directly with the production KSQL cluster.
+KSQL supports locked-down, "headless" deployment scenarios where interactive use of the KSQL cluster is disabled.
+For example, you want to allow a team of users to develop and verify their queries interactively on a shared testing
+KSQL cluster. But when putting those queries to production you prefer to lock-down access to KSQL Servers,
```

Comment:
`But when deploying those queries in your production environment, you want to lock-down access to KSQL Servers, version-control the exact queries, and store them in a .sql file. This will prevent users from interacting directly with the production KSQL cluster.`